.ci/Dockerfile.armv7l

```diff
@@ -0,0 +1,19 @@
+ARG image=balenalib/armv7hf-debian:buster
+FROM $image
+
+COPY . /kivy
+WORKDIR /kivy
+
+RUN [ "cross-build-start" ]
+
+# Install dependencies.
+RUN /bin/bash -c 'source .ci/ubuntu_ci.sh && \
```

Comment:
Rather than generating a wheel for the default Python version, could you maybe generate wheels for multiple python versions like at https://www.piwheels.org/project/Kivy/ and like we do for manylinux? It is ok to not do that if it's too complicated, it's just a thought.

Reply:
Fixed in: https://github.com/kivy/kivy/pull/6568/commits/e512e1f66a283ffd20536f81fe30204b4a9a5c17